diff --git a/setup.py b/setup.py
index 970e4fc..56b42b8 100644
--- a/setup.py
+++ b/setup.py
@@ -178,6 +178,7 @@ cmdclass = {'build_ext': build_ext}
 # distribution will be used.
 if HAVE_CYTHON:
     print ("# pysam: cython is available - using cythonize if necessary")
+    #
     source_pattern = "pysam/libc%s.pyx"
 else:
     print ("# pysam: no cython available - using pre-compiled C")
@@ -432,6 +433,7 @@ Operating System :: MacOS
 """
 
 metadata = {
+    'compiler_directives' : {'language_level' : sys.version_info[0]},
     'name': "pysam",
     'version': get_pysam_version(),
     'description': "pysam",
